httomo/loaders/standard_tomo_loader.py

```diff
@@ -0,0 +1,272 @@
+import weakref
+from pathlib import Path
+from typing import Literal, NamedTuple, Optional, Tuple, TypeAlias, Union
+
+import h5py
+import numpy as np
+from mpi4py import MPI
+
+from httomo.darks_flats import DarksFlatsFileConfig, get_darks_flats
+from httomo.preview import Preview, PreviewConfig
+from httomo.runner.dataset import DataSetBlock, FullFileDataSet
+from httomo.runner.dataset_store_interfaces import DataSetSource
+from httomo.runner.loader import LoaderInterface
+from httomo.utils import Pattern, log_once
+
+
+class RawAngles(NamedTuple):
+    data_path: str
+
+
+class UserDefinedAngles(NamedTuple):
+    start_angle: int
+    stop_angle: int
+    angles_total: int
+
+
+AnglesConfig: TypeAlias = Union[RawAngles, UserDefinedAngles]
```

Comment:
I think it's better to define the types where the interface to the outside world is. That is, I'd move this with `make_loader` - or have the types in a separate file. Then this file feels more like one possible implementation of a loader - and a future different one would not have to include this file to get the types.

Reply:
Good point, I wasn't sure where these types should live, I think alongside `make_loader()` in `httomo/loaders/__init__.py` is a good idea.

Reply:
I ended up using a separate file to hold the types, I got a circular import trying to put the types into `__init__.py`, see [e2170d0](https://github.com/DiamondLightSource/httomo/pull/209/commits/e2170d0fcd7ae123a7cc062cfb77553bf93e6d3a)